report_a_breach/templates/form_steps/summary.html

```diff
@@ -148,11 +148,20 @@ <h2 class="govuk-summary-card__title">About the suspected breach</h2>
                         Sanctions regimes breached
                     </dt>
                     <dd class="govuk-summary-list__value">
-                        N/A
+                        {% for regime in sanctions_regimes %}
```

Comment:
related to above, we can do

```suggestion
                        {% for regime in form_data.which_sanctions_regime.which_sanctions_regime %}
```